tests/TestCase/Cache/InvalidArgumentExceptionTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.4.12
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ * @deprecated Backwards compatibility test for alias. Will be removed in 5.0
+ */
+namespace Cake\Test\TestCase\Cache;
+
+use Cake\TestSuite\TestCase;
+
+/**
+ * InvalidArgumentExceptionTest class
+ */
+class InvalidArgumentExceptionTest extends TestCase
+{
+    /**
+     * tests That the old/deprecated class is properly aliased to the new one.
+     */
+    public function testThrowingOldException(): void
+    {
+        $this->expectException(\Cake\Cache\InvalidArgumentException::class);
```

Comment:
Use a `use` statement instead of the full path.

Reply:
Shouldn't cs fix this?

Reply:
I think it is failing, just trying to be helpful.

Reply:
No, thing is CI is green on that part
We might need a sniffer rule here

Reply:
The sniff for this exists in the our v5 ruleset, not v4.

Reply:
It was passing, but codecov was failing as it was looking coverage, that's why I added it. I can remove it.

Reply:
I removed it, and codecov dropped it's coverage percentage. Will that sniff in 5.x cover it?